client/app/src/main/java/com/microsoft/research/karya/injection/RetrofitModule.kt

```diff
@@ -22,7 +22,7 @@ class RetrofitModule {
 
   @Provides
   @Reusable
-  fun provideGsonConverterFactory(): GsonConverterFactory {
+  fun provide(): GsonConverterFactory {
```

Comment:
The old name made more sense, please rename it.

Reply:
sure